hedgehog/src/Hedgehog/Internal/State.hs

```diff
@@ -449,17 +449,14 @@ commandGenOK (CommandA inputGen _ _ _) state =
 --   evaluated.
 --
 data Action m (state :: (Type -> Type) -> Type) =
-  forall input0 input output.
+  forall input output.
   (TraversableB input, Show (input Symbolic), Show output) =>
   Action {
```

Comment:
Am I correct that we miss the name of the action here?
This would be purely for debugging purposes, but may make it easier to see what happens.